frontend/components/ItemViewer.vue

```diff
@@ -154,41 +159,38 @@ export default {
 
     orderClaims (claims) {
       const claimPs = Object.keys(claims)
-      claimPs.sort(function (x, y) {
```

Comment:
Now I understand, I think this point was not clear in the issue.

The idea is keep this code as is and get the array CLAIM_P_ORDER from the wikibase, in this case fetching the data from: 
https://pbuidev.wikibase.cloud/w/api.php?action=parse&page=MediaWiki:Wikibase-SortedProperties&prop=wikitext&formatversion=2&format=json&origin=*

You have an example of this kind of request here:
https://github.com/PhiloBiblon/philobiblon-ui/blob/master/frontend/pages/wiki/_page.vue#L30
but in this case we need to transform the `wikitext` to the CLAIM_P_ORDER (instead to be hardcoded).